subgraphs/_reference_/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "reference-subgraph",
+  "license": "UNLICENSED",
```

Comment:
just curious, what will be the license of these subgraphs?

Reply:
should be MIT